Blockzilla/AppDelegate.swift

```diff
@@ -70,6 +70,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, ModalDelegate, AppSplashC
         // We clear the Caches directory after each Erase, but WebKit apparently maintains
         // localStorage in-memory (bug 1319208), so we just disable it altogether.
         UserDefaults.standard.set(false, forKey: "WebKitLocalStorageEnabledPreferenceKey")
+        UserDefaults.standard.set(nil, forKey: "searchedHistory")
```

Comment:
Any difference between this and `UserDefaults.standard.removeObject(forKey: "searchedHistory")`?

Reply:
I think it would be preferable to use `UserDefaults.standard.removeObject(forKey: "searchedHistory")`

Reply:
@justindarc I'm pretty sure both will remove the key from UserDefaults.

Reply:
Ok cool. I wasn't sure if it would leave any remains or not. Its probably fine either way.

Reply:
I changed it just in case :)